HTML_To_Markdown.php

```diff
@@ -384,6 +384,8 @@ private function convert_anchor($node)
 
         if ($title != "") {
             $markdown = '[' . $text . '](' . $href . ' "' . $title . '")';
+        } elseif ($href === $text) {
+            $markdown = '<' . $text . '>';
```

Comment:
Would you mind changing this to use `$href` instead of `$text`?  I know it's an extremely minor nit-pick, but we are outputting a link after all, so using `$href` would make that super obvious.